game/project.godot

```diff
@@ -56,7 +56,7 @@ window/per_pixel_transparency/allowed=true
 
 [editor_plugins]
 
-enabled=PackedStringArray("res://addons/keychain/plugin.cfg", "res://addons/openvic-plugin/plugin.cfg")
+enabled=PackedStringArray("res://addons/MusicMeta/plugin.cfg", "res://addons/keychain/plugin.cfg", "res://addons/openvic-plugin/plugin.cfg")
```

Comment:
I'm going to see if there's godot way to fix this over git, because what actually happened was I switched from one plugin called MusicMeta to another which is more active/recent called MusicMetadata (rather than the folder being renamed). That or the plugin devs made a mistake.